charts/stable/nextcloud/values.yaml

```diff
@@ -1,12 +1,12 @@
 image:
   repository: tccr.io/truecharts/nextcloud-fpm
   pullPolicy: IfNotPresent
-  tag: v24.0.2@sha256:491aa04ba52e5fe8f48e2efbe544af263fd13fc41d1793a830aadd5e7721cb0d
+  tag: v24.0.3@sha256:5cc5f0d086e4ead89c1988f1cd4d9f6820cac63ac2647a012c46aab45fc9c0a0
```

Comment:
I think this broke things? https://github.com/truecharts/containers/runs/7423404407?check_suite_focus=true
```
 "sources": [
        {
          "type": "docker-image",
          "ref": "docker.io/library/nextcloud:24.0.3-fpm@sha256:c4c67e2b00f911083dc6aea03ef5e5e403001d0c3a0d1535683c08594c9e702d",
          "pin": "sha256:c4c67e2b00f911083dc6aea03ef5e5e403001d0c3a0d1535683c08594c9e702d"
        }
      ]
      ```
      
      And then on my nextcloud install:
      ```
      Error syncing pod, skipping" err="failed to \"StartContainer\" for \"nextcloud\" with ImagePullBackOff: \"Back-off pulling image \\\"tccr.io/truecharts/nextcloud-fpm:v24.0.3@sha256:5cc5f0d086e4ead89c1988f1cd4d9f6820cac63ac2647a012c46aab45fc9c0a0\\\"\"" pod="ix-nextcloud/nextcloud-cronjob-27640940-sgxdj" podUID=d6375014-ed96-44a1-939f-bb8f224cef32
```

Or am I missing something?